.circleci/config.yml

```diff
@@ -418,9 +418,20 @@ jobs:
           name: Build
           command: NODE_OPTIONS="--max-old-space-size=1610" bash -i -c 'npm run build.prod'
 
+parameters:
+  run_nightly_auth:
+    type: boolean
+    default: false
+  run_nightly_no_auth:
+    type: boolean
+    default: false
+
 workflows:
   version: 2
   everything:
+    when:
+      not:
+        equal: [ scheduled_pipeline, << pipeline.trigger_source >> ]
```

Comment:
Cool!  One option here might be to instead check for the condition `! ( pipeline.parameters.run_nightly_no_auth || pipeline.parameters.run_nightly_auth )`.  That way, you could schedule the full run too, if you ever wanted to.

Reply:
done!